cli_tools/gce_ovf_import/ovf_utils/ovf_utils.go

```diff
@@ -35,42 +35,169 @@ const (
 	usbController          uint16 = 23
 )
 
-//Full list: https://vdc-download.vmware.com/vmwb-repository/dcr-public/da47f910-60ac-438b-8b9b-6122f4d14524/16b7274a-bf8b-4b4c-a05e-746f2aa93c8c/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
+type osInfo struct {
+	description      string
+	importerOSIDs    []string
+	nonDeterministic bool
```

Comment:
Please add a comment for what the flag signifies. Makes it easier to understand intended behavior of flags with comments

Reply:
Agree. I thought len(importerOSIDs)>1 means nonDeterministic, however after I read the whole PR I saw they are different. Would you explain why it's nonDeterministic when it's a 1:1 map?

Reply:
Added comments in the code, but the summary is that there are two main reasons:
1. Legal: we don't want to assume BYOL for Windows client because we don't know if a customer has BYOL agreement.
2. Possible multiple mappings but we only support one in import, so again we don't want to assume it's the version we support (e.g. "Suse 64-bit" to `opensuse-15` because there are other versions of Suse).